src/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  overrides: [
+    {
+      files: [
+        '**/*.{cjs,js,mjs}'
```

Comment:
What shall we do about this bit?

Think we should either:

1. Use `"**/*.{cjs,js,mjs}"` everywhere (default ESLint file extensions)
2. Use `"**/*.js"` everywhere (removes `*.cjs` and `*.mjs` from being matched)

(The **npm run lint:js** script only includes `**/*.js`)

Reply:
I've gone with including `"**/*.{cjs,js,mjs}"` everywhere (thanks for catching the missed bit in the script), for future-proofing.